test/errors/src/index.md

```diff
@@ -34,4 +48,3 @@ This is the footnote [^1]. And [^another] [^another].
 [^nested]:
 
     Duplicate [^1] nested footnote.
-
```

Comment:
Unrelated changes shouldn't be included. Especially whitespace ones. We can look into formatting everything in a consistent way in a separate PR if you'd like to (and have the time).